Source/Swig/typemap.c

```diff
@@ -1967,8 +1967,9 @@ static void replace_embedded_typemap(String *s, ParmList *parm_sublist, Wrapper
 	    String *parm = Getitem(l, i);
 	    char *eq = strchr(Char(parm), '=');
 	    char *c = Char(parm);
-	    if (eq && (eq - c > 0)) {
-	      String *name = NewStringWithSize(c, (int)(eq - c));
+	    int size = (int)(eq - c);
+	    if (eq && size > 0) {
+	      String *name = NewStringWithSize(c, size);
```

Comment:
This change seems to introduce undefined behaviour in the case where `eq == NULL`.